compiler/next/include/chpl/queries/Context.h

```diff
@@ -313,20 +313,11 @@ class Context {
    */
   void markUniqueCString(const char* s);
 
-  /**
-    Return the name of the module containing this ID.
-   */
-  UniqueString moduleNameForID(ID id);
   /**
     Return the file path for the file containing this ID.
    */
   UniqueString filePathForID(ID id);
 
-  /**
-    Query to get a file path given a module name
-   */
-  UniqueString filePathForModuleName(UniqueString modName);
```

Comment:
It's replaced by filePathForID now that modules have better behaving IDs.

The problem with this function is that, in the case of a nested module, just knowing the module name isn't enough to know the path to the file it came from. (It is enough for a top-level module because we track that and there can't be duplicate-ly named top-level modules). So I just removed it instead of renaming it to e.g. filePathForTopLevelModuleName and not using it anywhere :)